openapi-v1.yaml

```diff
@@ -4372,3 +4425,64 @@ paths:
           description: error response
           schema:
             $ref: "#/definitions/Error"
+
+  /favorites:
+    get:
+      summary: Your GET endpoint
+      tags:
+        - favorites
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Favorite'
+      operationId: get-favorites
+      parameters:
+        - type: string
+          in: query
+          name: type
+          description: The type of the desired favorites e.g. arrays
+    parameters: []
+    post:
+      summary: ''
+      operationId: post-favorite
+      responses:
+        '200':
+          description: OK
+      description: Add a new favorite
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: '#/definitions/FavoriteUpdate'
+      tags:
+        - favorites
+  '/favorites/{id}':
+    get:
+      summary: Your GET endpoint
+      tags:
+        - favorites
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Favorite'
+      operationId: get-favorite
+      description: ''
+    parameters:
+      - type: string
+        name: id
+        in: path
+        required: true
+        description: The uuid of the favorite
+    delete:
+      summary: ''
+      operationId: delete-favorite-id
```

Comment:
Please drop the `-`, as this can cause problems with code generation. Use either `_` or camel case.